src/components/Editor.js

```diff
@@ -42,26 +42,6 @@ function breakpointAtLine(breakpoints, line) {
   });
 }
 
-function getTextForLine(codeMirror, line) {
-  return codeMirror.getLine(line - 1).trim();
-}
-
-function getCursorLine(codeMirror) {
-  return codeMirror.getCursor().line;
-}
-/**
- * Forces the breakpoint gutter to be the same size as the line
- * numbers gutter. Editor CSS will absolutely position the gutter
- * beneath the line numbers. This makes it easy to be flexible with
- * how we overlay breakpoints.
- */
-function resizeBreakpointGutter(editor) {
-  const gutters = editor.display.gutters;
-  const lineNumbers = gutters.querySelector(".CodeMirror-linenumbers");
-  const breakpoints = gutters.querySelector(".breakpoints");
-  breakpoints.style.width = `${lineNumbers.clientWidth}px`;
-}
```

Comment:
extracting these three utilities is the right idea. Unfortunately, moving them to the SourceEditor won't work because we have a different SourceEditor in the panel :/

My vote would be to move these utils to a new util file in a later PR. Infact, i think there are other utilites we can extract as well :)